app/server/appsmith-interfaces/src/main/java/com/appsmith/external/annotations/documenttype/DocumentType.java

```diff
@@ -1,4 +1,4 @@
-package com.appsmith.external.annotations;
+package com.appsmith.external.annotations.documenttype;
```

Comment:
Super minor: Suggestion to use Camel case. Keeps it consistent.
```suggestion
package com.appsmith.external.annotations.documentType;
```

Reply:
IIRC, the convention is to use lower case for package names always. Since package names are folder names and not all operating systems use case sensitive file names.

Reply:
Aah sorry. My bad. Resolving this comment.